tests/spec/RulerZ/Parser/ParserSpec.php

```diff
@@ -64,6 +64,7 @@ public function invalidRules()
             '> 30',
             'name[0] = "a"',
             'name.foo() = "a"',
+            'foo = ',
```

Comment:
added because it was the only code block for Rusty that was actually asserting something, see https://github.com/wallabag/rulerz/blob/867e89e33fc3017e9539b9d3250102c888bd8635/src/Parser/Parser.php#L24